hana/exporter.sls

```diff
@@ -5,7 +5,8 @@
 {% if node.host == host and node.exporter is defined %}
 
 {% set instance = '{:0>2}'.format(node.instance) %}
-{% set config_file = '/etc/hanadb_exporter/config_{}_{}.json'.format(node.sid, instance) %}
+{% set daemon_instance = '{}_{}'.format(node.sid, instance) %}
```

Comment:
I personally prefer `{0} {1}` it make the code imho more implicit. But is just a personal style, ok with both.

Reply:
@MalloZup Do you mean `{0}_{1}`? As this extension will be used to match with a file name, we cannot have spaces.

Reply:
@arbulu89 I mean the format thing in python. you can write `{0}` or `{}` they are identic. 

but is same.. :grin:

Reply:
I'm resolving now